Moonlight macOS/Utility/keyboardTranslation.m

```diff
@@ -0,0 +1,163 @@
+//
+//  keyboardTranslation.m
+//  Moonlight macOS
+//
+//  Created by Felix Kratz on 10.03.18.
+//  Copyright © 2018 Felix Kratz. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "keyboardTranslation.h"
+#import <Limelight.h>
+
+CGKeyCode modifierKeyFromEvent(int keyModifier) {
+    switch (keyModifier) {
+        case 131330:
```

Comment:
Surely there are `const`s or `#define`s for these

Reply:
I have used the const's for now.